metals/src/main/scala/scala/meta/internal/metals/MetalsEnrichments.scala

```diff
@@ -361,4 +361,19 @@ object MetalsEnrichments extends DecorateAsJava with DecorateAsScala {
 
   }
 
+  implicit class XtensionDocumentSymbol(documentSymbol: l.DocumentSymbol) {
+    def toSymbolOccurrence: s.SymbolOccurrence = s.SymbolOccurrence(
```

Comment:
Thinking out loud. This seems only used in tests. I suspect the same applies for other methods in this file so it's not a big deal. Maybe we should have a similar utility file in tests?

Reply:
Done (only for `XtensionDocumentSymbol` right now). We can use to dump other test-only extensions.